highfidelity/hifi
# case 19562: GLES - Use TBO instead of SSBO and make shaders use it when it is not gl45 (master)

https://highfidelity.manuscript.com/f/cases/19562/Avatars-do-not-load-in-Android

Same fix as this pr #14370

with the fix for the mac

----

Rollback GLES to work as GL41 does as those storage blocks are not mandatory in GLES implementations/devices.

**Test plan**

1. Launch the app in Android
2. Enter any domain or last location
3. My Avatar should render (before this PR it isn't happening).
4. Windows/Linux/macOS rendering shouldn't be affected, please check.

**Note**
On some Mali GPU Devices (Like Huawei Mate 9 Pro - Use https://play.google.com/store/apps/details?id=de.saschawillems.glescapsviewer&hl=en app to confirm, OpenGL ES - Vendor and Renderer) Interface may flicker being unusable, that's a different issue not addressed in this PR.
